pkg/tcpip/iptables/iptables.go

```diff
@@ -34,9 +34,9 @@ const (
 // all packets.
 func DefaultTables() *IPTables {
 	tables := IPTables{
```

Comment:
Just return &IPTables{ ?

It looks like storing it in a named variable is no longer required.